napari/layers/shapes/shapes.py

```diff
@@ -1574,6 +1575,283 @@ def _set_editable(self, editable=None):
         if not self.editable:
             self.mode = Mode.PAN_ZOOM
 
+    def add_rectangles(
+        self,
+        data,
+        *,
+        edge_width=None,
+        edge_color=None,
+        face_color=None,
+        z_index=None,
+    ):
+        """Add rectangles to the current layer.
+
+        Parameters
+        ----------
+        data : Array | List[Array]
+            List of rectangle data where each element is a (4, D) array of 4 vertices
```

Comment:
Single data (ie a single (2, D) array) are allowed, right? I feel like the docstring should say that...

Reply:
Updated all docstrings to reflect this usage.